cmd/deploy_delete.go

```diff
@@ -23,8 +23,15 @@ func newDeleteDeploymentCmd(deployConfig *deployCommandConfig) *cobra.Command {
 	var deployConfigFile string
 	var deleteDeploymentCmd = &cobra.Command{
 		Use:   "delete",
-		Short: "Delete your deployed Appsody project from a Kubernetes cluster",
-		Long:  `This command deletes your deployed Appsody project from the configured Kubernetes cluster using your existing deployment manifest.`,
+		Short: "Delete your deployed Appsody project from a Kubernetes cluster.",
+		Long: `Delete your deployed Appsody project from the configured Kubernetes cluster using your existing deployment manifest.
+
+By default, the command looks for the deployed project in the "default" namespace, and uses the generated "app-deploy.yaml" deployment manifest, unless you specify otherwise.`,
+		Example: `  appsody deploy delete -f my-deploy.yaml
+  Deletes the AppsodyApplication from the "default" namespace using the name and type specified in the "my-deploy.yaml" deployment manifest.
+  
+  appsody deploy delete --namespace my-namespace
+  Deletes the AppsodyApplication from the "my-namespace" namespace using the name and type specified in the "app-deploy.yaml" deployment manifest`,
```

Comment:
Same comments as earlier, plus needs a full stop at the end:
"Deletes the Appsody application from the "my-namespace" namespace, using the name and type specified in the "app-deploy.yaml" deployment manifest."